Solution1/Solution1.Tests/BL/Services/WeatherApiServiceTests.cs

```diff
@@ -45,10 +44,10 @@ public async Task GetByCityNameAsync_ReturnedWeatherApiDTO_Success()
                 Content = new StringContent(JsonSerializer.Serialize(new { Main = new { Temp = temp }, Name = cityName }, _serializerOptions)),
             };
 
-            SetHttpHandlerSettings(_httpMessageHandler, response, urlString);
+            SetHttpHandlerSettings(response, urlString);
 
             // Act
-            var result = await _weatherApiService.GetByCityNameAsync(cityName);
+            var result = await _weatherApiService.GetByCityNameAsync(cityName, new CancellationToken());
```

Comment:
CancellationToken.None when you don't need smth specific from it

Reply:
+

Reply:
?